src/features/src/muteFeature.tsx

```diff
@@ -0,0 +1,33 @@
+import FFmpegFeature from '../FFmpegFeature'
+
+class MuteFeature extends FFmpegFeature {
+  configuration = {}
+
+  constructor(configuration = {}) {
+    super()
+    this.configuration = configuration
+
+    this.setProgress()
+    this.setFileConfig()
+    this.setFFmpegCommands()
+  }
+
+  parseConfiguration = () => ({ noConfig: true })
+
+  setProgress = () => {
+    this.progressBar = { name: 'Muting Video ...', color: 'black' }
+  }
+
+  setFileConfig = () => {
+    this.fileConfig = {
+      primaryType: 'video',
+      types: [{ name: 'video', number: { min: 1, max: 1 } }]
+    }
+  }
+
+  setFFmpegCommands = () => {
+    this.ffmpegCommands = '-an'
+  }
```

Comment:
This is neat, really clean way to mute audio files sometime FFmpeg has really smart things